package.json

```diff
@@ -98,7 +98,7 @@
     "webpack-bundle-analyzer": "^4.5.0"
   },
   "engines": {
-    "node": "^16.8.0 || ^18.0.0 || ^19.0.0 || ^20.0.0 || ^21.0.0"
+    "node": "^16.8.0 || ^18.0.0 || ^19.0.0 || ^20.0.0 || ^21.0.0 || ^22.0.0"
```

Comment:
at this point we should actually not tag any versions but rather have node to be `>=16.8.0` is what I feel? @rickhanlonii 

similar to https://github.com/vercel/next.js/blob/cc10bf5d1fa4fd2bead34661070c936d360516b3/packages/next/package.json#L340

Reply:
I can update it like this if you want.

Reply:
We might also want to verify if everything is working fine before we add these node versions. Like the start scripts. Seems like 17 alone is missing in between. If its verified we are GTG.

Reply:
Yes please, I loath node version checks - let's verify the oldest supported version and let's just be `>=` to that version, likely the same version Next requires. In the future, if a newer node version is broken, we can try to change the APIs so we support the newer version.